docs/firestore/reference/Settings.md

```diff
@@ -14,6 +14,13 @@ The hostname to connect to.
 
 Whether to use persistence.
 
+### cacheSizeBytes
+[method]cacheSizeBytes returns nullable number;[/method]
+
+Sets an approximate cache size threshold for the on-disk data. If the cache grows beyond this size, Firestore will start removing data that hasn't been recently used. The size is not a guarantee that the cache will stay below that size, only that if the cache exceeds the given size, cleanup will be attempted.
+
```

Comment:
Should we mention the default size of 100mb and any min / max sizes to match the validation?

Reply:
A good idea, I have some other doc updates to do before release, I'll handle it from here @mgenov has done excellent work by all accounts and I'll be in the docs anyway

Reply:
Added parapgraph for the default size, min and unlimited.